capstone-frontend/app/screens/SignInScreen.js

```diff
@@ -23,13 +25,20 @@ export default function SignInScreen() {
     }
   }
 
+  
+    
     const handleOnChange = (text,input) => {
       setInputs(prevState=>({...prevState, [input]: text}))
+      
     }
 
-    const handleError = (errorMessage,input)=>{
-      setErrors(prevState=>({...prevState, [input]: errorMessage}))
-    }
+    const handleSubmit=(e)=>{
```

Comment:
What is the use of this method, there is already a method present to handle submit